search.js

```diff
@@ -98,12 +100,12 @@ function main(params) {
             x = `metadata.sidebar:"${x}"`;
           });
           filter = `(${sidebars.join('|')})`;
-          data.filter = filter;
+        // data.filter = filter;
```

Comment:
is it intended to ignore the filter?

Reply:
I wasn't able to get the search to return anything if this line is running. But currently we have the problem that it always return you the LB4 results first even you're searching from a LB3 page.

Reply:
Looks like the previous way to generate a `posts.json` file for the data content has deprecated:
https://cloud.ibm.com/docs/services/discovery?topic=discovery-getting-started-with-the-data-crawler. I think the filtering by sidebar value no longer works for us.  I'll update the PR. Thanks.